gui/src/2D/controls/scrollViewers/scrollViewer.ts

```diff
@@ -690,11 +687,7 @@ export class ScrollViewer extends Rectangle {
 
     /** Releases associated resources */
     public dispose() {
-        let scene = this._host.getScene();
-        if (scene && this._onPointerObserver) {
-            scene.onPointerObservable.remove(this._onPointerObserver);
-            this._onPointerObserver = null;
-        }
+        this._onPointerObserver = null;

```

Comment:
you need to remove the observer from the observable

Reply:
Done and thanks!